src/Util/PHP/AbstractPhpProcess.php

```diff
@@ -209,6 +209,10 @@ public function getCommand(array $settings, string $file = null): string
                     array_keys(ini_get_all('xdebug'))
                 )
             );
+
+            if (!$this->runtime->isXdebugLoadedByDefault()) {
+                $settings[] = 'zend_extension=' . (getenv('TODO_USE_PHPUNIT_SETTINGS') ?: 'php_xdebug');
```

Comment:
getenv('TODO_USE_PHPUNIT_SETTINGS')
    
 Should I...
1. ...just use `php_xdebug` since it already solves most cases
2. ...use `getenv('XX')` to offer a bit of flexibility (what should it be named?)
3. ...use PHPUnit xml config functionality to "do it properly"